cw20/base/.github/workflows/Basic.yml

```diff
@@ -64,20 +64,12 @@ jobs:
           command: clippy
           args: -- -D warnings
 
-      - name: Generate schema
+      - name: Generate Schema
         uses: actions-rs/cargo@v1
         with:
           command: schema
           args: --locked
 
-      - name: Verify schema
-        uses: tj-actions/verify-changed-files@v8
-        id: verify-schema
-        with:
-          files: schema/.*\.json
-
-      - name: Display changed schemas
-        if: steps.verify-schema.outputs.files_changed == 'true'
-        run: |
-          echo "The schema files are not in sync with the repository. Please, run 'cargo schema' to generate them again and commit the changes."
-          exit 1
```

Comment:
Please revert this change. I added the message to clarify how to fix it for the users.

Reply:
@gachouchani1999 this is still not resolved. You moved this exact code to the wrong place in the file. Please, recheck it.